WikipediaLibrary/externallinks
# User list filtering

## Description
In [T269147](https://phabricator.wikimedia.org/T269147) we reworked how Program and Organisation page load data, using the new aggregates tables. This required temporarily removing the user list filtering functionality. 

This has been implemented again in this PR.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
Filtering by user list is a crucial feature so that we can distinguish which link additions were made by editors who have access through The Wikipedia Library.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T269841](https://phabricator.wikimedia.org/T269841)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
Manual testing in Program and Organisation details pages. 

1. Navigate to a Program or Organisation detail page
2. Select the `Limit to user list` option and click on `Submit`
3. Check that the information is filtered by users on the Wikipedia Library user list

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)
![Screen Shot 2021-01-05 at 19 14 03](https://user-images.githubusercontent.com/7854953/103717088-36e52480-4f8a-11eb-91ed-b307e4889040.png)


## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
